laske_export/document/custom_validators.py

```
@@ -0,0 +1,39 @@
+from django.core.exceptions import ValidationError
+
+
+def calculate_checksum(value: str | None) -> str | None:
+    """
+    Calculates the checksum for digits of the payment reference (excluding the last digit)
+    """
+    if not value:
+        return None
```

I would also raise a validation error here. I think the resulting calculated checksum from this function must always be the checksum digit. None might create weird errors in the future, even if the current flow doesn't have the possiblity for such errors.